src/main/java/com/farao_community/farao/core_cc/adapter/service/CoreCCAdapterService.java

```diff
@@ -188,4 +190,19 @@ private void addNewRunInTaskHistory(OffsetDateTime timestamp, List<ProcessFileDt
         final String url = taskManagerTimestampBaseUrl + timestamp + "/runHistory";
         restTemplateBuilder.build().put(url, inputFiles);
     }
+
+    private TaskDto getUpdatedTask(OffsetDateTime timestamp) {
```

Comment:
Maybe move this method so that it is grouped with `updateTaskStatusToPending()` and `addNewRunInTaskHistory()` which are the two other method that perform a call to the task-manager.
I would even suggest that these three methods be extracted to a new service class, but maybe you will find it too extreme.